src/momento/internal/aio/_scs_control_client.py

```diff
@@ -73,38 +88,37 @@ async def create_signing_key(self, ttl_minutes: int, endpoint: str) -> CreateSig
             create_signing_key_request = _CreateSigningKeyRequest()
             create_signing_key_request.ttl_minutes = ttl_minutes
             return CreateSigningKeyResponse.from_grpc_response(
-                await self._grpc_manager.async_stub().CreateSigningKey(
-                    create_signing_key_request, timeout=_DEADLINE_SECONDS
-                ),
+                await self._getStub().CreateSigningKey(create_signing_key_request, timeout=_DEADLINE_SECONDS),
                 endpoint,
             )
         except Exception as e:
             self._logger.warning(f"Failed to create signing key with exception: {e}")
-            raise _cache_service_errors_converter.convert(e)
+            raise convert_error(e)
 
     async def revoke_signing_key(self, key_id: str) -> RevokeSigningKeyResponse:
         try:
             self._logger.info(f"Revoking signing key with key_id {key_id}")
             request = _RevokeSigningKeyRequest()
             request.key_id = key_id
-            await self._grpc_manager.async_stub().RevokeSigningKey(request, timeout=_DEADLINE_SECONDS)
+            await self._getStub().RevokeSigningKey(request, timeout=_DEADLINE_SECONDS)
             return RevokeSigningKeyResponse()
         except Exception as e:
             self._logger.warning(f"Failed to revoke signing key with key_id {key_id} exception: {e}")
-            raise _cache_service_errors_converter.convert(e)
+            raise convert_error(e)
 
     async def list_signing_keys(self, endpoint: str, next_token: Optional[str] = None) -> ListSigningKeysResponse:
         try:
             list_signing_keys_request = _ListSigningKeysRequest()
             list_signing_keys_request.next_token = next_token if next_token is not None else ""
             return ListSigningKeysResponse.from_grpc_response(
-                await self._grpc_manager.async_stub().ListSigningKeys(
-                    list_signing_keys_request, timeout=_DEADLINE_SECONDS
-                ),
+                await self._getStub().ListSigningKeys(list_signing_keys_request, timeout=_DEADLINE_SECONDS),
                 endpoint,
             )
         except Exception as e:
-            raise _cache_service_errors_converter.convert(e)
+            raise convert_error(e)
+
+    def _getStub(self) -> ScsControlStub:
```

Comment:
I changed some camel cased function names in my PR to reduce IDE warning noise and for general consistency, including a `_getStub()` in another file. Would you mind changing this one to `_get_stub()` as well, please.

Reply:
Fixed in [2e1ad18](https://github.com/momentohq/client-sdk-python/pull/179/commits/2e1ad183aa54984e7fc9d1a43388e2da14b8069e). I changed the name to `_build_stub` to highlight we are creating a new instance as opposed to getting the same instance.